src/grun-maciej/src/Fsd.Maciej.Cs/Fsd.Maciej.Cs.Ex2/Factory/CowFactory.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Fsd.Maciej.Cs.Ex2
+{
+    class CowFactory
```

Comment:
identyczne uwagi jak dla CatFactory - nie powinno być powielenia tego kodu. Jest to złamanie zasady Open/Closed